source/grid/tria.cc

```diff
@@ -1066,7 +1066,6 @@ namespace internal
       {
         (void)global_first_cell;
         (void)global_num_cells;
```

Comment:
It would be great to address both of these variables while you are at it :)

Reply:
I was suppose to do only 5-10, other people are doing the other ones. There are actually 23 occurence of (void)x in this file.

Reply:
Okidoke :) Sorry about that it's my bad.